### PR TITLE
Cave Veins render correctly now

### PR DESCRIPTION
**Pull request by mMONTAGEe**
(dont mind the mess i still dont knwo how to correctly make multiply PRs)

### CODE REVIEW

File: src/main/java/dev/galacticraft/mod/GalacticraftClient.java [hunk 1]
```diff
@@ -24,12 +24,12 @@
 
 import dev.galacticraft.mod.block.GalacticraftBlock;
 import dev.galacticraft.mod.client.gui.screen.ingame.*;
-import dev.galacticraft.mod.client.model.MachineBakedModel;
```
Comment (craftablescience) [inline src/main/java/dev/galacticraft/mod/GalacticraftClient.java:27]: Can you remove all the unrelated changes in the file?

File: src/main/java/dev/galacticraft/mod/GalacticraftClient.java [hunk 2]
```diff
@@ -44,7 +44,7 @@
 import net.fabricmc.fabric.api.blockrenderlayer.v1.BlockRenderLayerMap;
 import net.fabricmc.fabric.api.client.model.ModelLoadingRegistry;
 import net.fabricmc.fabric.api.client.particle.v1.ParticleFactoryRegistry;
-import net.fabricmc.fabric.api.client.rendereregistry.v1.EntityRendererRegistry;
```
Comment (craftablescience) [inline src/main/java/dev/galacticraft/mod/GalacticraftClient.java:47]: Ditto

File: src/main/java/dev/galacticraft/mod/GalacticraftClient.java [hunk 3]
```diff
@@ -60,7 +60,6 @@
  */
 @Environment(EnvType.CLIENT)
 public class GalacticraftClient implements ClientModInitializer {
-
```
Comment (craftablescience) [inline src/main/java/dev/galacticraft/mod/GalacticraftClient.java:63]: Ditto

File: src/main/java/dev/galacticraft/mod/GalacticraftClient.java [hunk 4]
```diff
@@ -107,32 +106,34 @@ public void onInitializeClient() {
         ScreenRegistry.register(GalacticraftScreenHandlerType.OXYGEN_STORAGE_MODULE_HANDLER, OxygenStorageModuleScreen::new);
         ScreenRegistry.register(GalacticraftScreenHandlerType.OXYGEN_SEALER_HANDLER, OxygenSealerScreen::new);
 
-        EntityRendererRegistry.INSTANCE.register(GalacticraftEntityType.MOON_VILLAGER, (dispatcher, context) -> new MoonVillagerEntityRenderer(dispatcher, context.getResourceManager()));
```
Comment (craftablescience) [inline src/main/java/dev/galacticraft/mod/GalacticraftClient.java:110]: Ditto

File: src/main/java/dev/galacticraft/mod/GalacticraftClient.java [hunk 5]
```diff
@@ -107,32 +106,34 @@ public void onInitializeClient() {
...
+        EntityRendererRegistry.register(GalacticraftEntityType.EVOLVED_SPIDER, EvolvedSpiderEntityRenderer::new);
+        EntityRendererRegistry.register(GalacticraftEntityType.EVOLVED_EVOKER, EvolvedEvokerEntityRenderer::new);
+        EntityRendererRegistry.register(GalacticraftEntityType.EVOLVED_PILLAGER, EvolvedPillagerEntityRenderer::new);
+        EntityRendererRegistry.register(GalacticraftEntityType.EVOLVED_VINDICATOR, EvolvedVindicatorEntityRenderer::new);
+        EntityRendererRegistry.register(GalacticraftEntityType.BUBBLE, BubbleEntityRenderer::new);
 
         GalacticraftBlockEntityRenderer.register();
         GalacticraftClientPacketReceiver.register();
         MachineBakedModel.registerDefaults();
+        GalacticraftEntityModelLayer.register();
```
Comment (craftablescience) [inline src/main/java/dev/galacticraft/mod/GalacticraftClient.java:121]: Ditto

File: src/main/java/dev/galacticraft/mod/GalacticraftClient.java [hunk 6]
```diff
@@ -107,32 +106,34 @@ public void onInitializeClient() {
...
 
         BlockRenderLayerMap.INSTANCE.putBlock(GalacticraftBlock.TIN_LADDER, RenderLayer.getCutout());
-        BlockRenderLayerMap.INSTANCE.putBlock(GalacticraftBlock.GLASS_FLUID_PIPE, RenderLayer.getTranslucent());
+        BlockRenderLayerMap.INSTANCE.putBlock(GalacticraftBlock.GLASS_FLUID_PIPE, RenderLayer.getCutout());
```
Comment (craftablescience) [inline src/main/java/dev/galacticraft/mod/GalacticraftClient.java:124]: Ditto

File: src/main/java/dev/galacticraft/mod/GalacticraftClient.java [hunk 7]
```diff
@@ -107,32 +106,34 @@ public void onInitializeClient() {
...
         BlockRenderLayerMap.INSTANCE.putBlock(GalacticraftBlock.WALKWAY, RenderLayer.getCutout());
         BlockRenderLayerMap.INSTANCE.putBlock(GalacticraftBlock.WIRE_WALKWAY, RenderLayer.getCutout());
-        BlockRenderLayerMap.INSTANCE.putBlock(GalacticraftBlock.PIPE_WALKWAY, RenderLayer.getTranslucent());
+        BlockRenderLayerMap.INSTANCE.putBlock(GalacticraftBlock.PIPE_WALKWAY, RenderLayer.getCutout());
```
Comment (craftablescience) [inline src/main/java/dev/galacticraft/mod/GalacticraftClient.java:127]: Ditto

File: src/main/java/dev/galacticraft/mod/GalacticraftClient.java [hunk 8]
```diff
@@ -142,6 +143,16 @@ public void onInitializeClient() {
         ModelLoadingRegistry.INSTANCE.registerResourceProvider(resourceManager -> (resourceId, context) -> {
             if (MachineBakedModel.MACHINE_MARKER.equals(resourceId)) {
                 return MachineUnbakedModel.INSTANCE;
+            } else if (WireBakedModel.WIRE_MARKER.equals(resourceId)) {
```
Comment (craftablescience) [inline src/main/java/dev/galacticraft/mod/GalacticraftClient.java:146]: Ditto

**Review by craftablescience — CHANGES_REQUESTED** [review]

File: src/main/java/dev/galacticraft/mod/GalacticraftClient.java (hunk 1 above)
Comment (mMONTAGEe) [reply src/main/java/dev/galacticraft/mod/GalacticraftClient.java:27]: one sec

**Review by craftablescience — APPROVED** [review]
